README.md

```diff
@@ -129,6 +129,20 @@ Open index.html using live server extension.
 - GitHub: [IsaMarvin](https://github.com/IsaMarvin)
 - LinkedIn: [IsaMarvin](https://www.linkedin.com/in/isamarvin/)
```

Comment:
- [ ] This highlighted link for your LinkedIn profile is currently broken. Please kindly provide the appropriate link to your readme so that it looks very professional 👍 ✨